index.html

```diff
@@ -65,287 +65,21 @@ <h2>I'm a Software Developer</h2>
-                <div class="project-section second desk-btn">
-                    <div class="project-section-box">
-                        <h2>Data Dashboard <br>
-                            Healthcare
-                        </h2>
-                        <p>
-                            A daily selection of privately <br>
-                            personalized reads; no accounts or <br>
-                            sign-ups required. Has been the <br>
-                            industry's standard.
-                        </p>
-                        <div class="btn">
-                            <ul>
-                                <li>HTML</li>
-                                <li>Bootstrap</li>
-                                <li>Ruby</li>
-                            </ul>
-                        </div>
-                        <div class="see-btn">
-                            <button>See Projects</button>
-                        </div>
-                    </div>
-                </div>
-                <div class="project-section third desk-btn">
-                    <div class="project-section-box">
-                        <h2>Website Portfolio</h2>
-                        <p>
-                            A daily selection of privately <br>
-                            personalized reads; no accounts or <br>
-                            sign-ups required. Has been the <br>
-                            industry's standard.
-                        </p>
-                        <div class="btn">
-                            <ul>
-                                <li>HTML</li>
-                                <li>Bootstrap</li>
-                                <li>Ruby</li>
-                            </ul>
-                        </div>
-                        <div class="see-btn">
-                            <button>See Projects</button>
-                        </div>
-                    </div>
 
-                </div>
-                <div class="project-section desk-btn">
-                    <div class="project-section-box">
-                        <h2>Profesional Art <br>
-                            Printing Data
-                        </h2>
-                        <p>
-                            A daily selection of privately <br>
-                            personalized reads; no accounts or <br>
-                            sign-ups required. Has been the <br>
-                            industry's standard.
-                        </p>
-                        <div class="btn">
-                            <ul>
-                                <li>HTML</li>
-                                <li>Bootstrap</li>
-                                <li>Ruby</li>
-                            </ul>
-                        </div>
-                        <div class="see-btn">
-                            <button>See Projects</button>
-                        </div>
-                    </div>
```

Comment:
- [ ] The project popup that you have implemented does not match the Figma Design. 

your work | Figma Design
------- | -----
![image](https://github.com/Blessman-Newton/Portfolio/assets/84156714/17f52180-6a33-4130-bd06-d670e1ac7180) | ![image](https://github.com/Blessman-Newton/Portfolio/assets/84156714/e8e631c8-b11b-40c8-ae3c-7de21e4fea2f)


As you would notice above that some portions of the popup are hidden like the buttons and some text. this is majorly due to the fact that you have not implemented **_scroll to view._**, the modal/popup container should have `overflow: scroll`

On desktop view, it is the same issue but its time the buttons overflow the modal container. Ser the `overflow:scroll` and this will fix it. You can google the actual solution that will work in case my suggested code does not solve it.

<img width="877" alt="image" src="https://github.com/Blessman-Newton/Portfolio/assets/84156714/a0b629a3-69fd-40a1-a650-5eb8d1f4cf08">

Reply:
You did well in making the changes suggested by the previous reviewer, kindly add some margin-bottom to your desktop view, and make sure it has a full view so as to enable you have a more presentable display 😄 

Your view :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/f700cdee-8102-4d01-b86d-1e7640250159)

Reply:
Kindly note that the mobile view has some padding issues especially to the right, could you please fix it 😄 
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/f3b53a56-c0b3-43e5-9d5b-2c6f8b2b9349)